Tests/NIOHTTPTests/HTTPDecoderLengthTest.swift

```diff
@@ -0,0 +1,556 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import NIOCore
+import NIOEmbedded
+import NIOHTTP1
+
+private class MessageEndHandler<Head: Equatable, Body: Equatable>: ChannelInboundHandler {
```

Comment:
Presumably you don't actually want to add this.

Reply:
removed!